docs/toast.html

```diff
@@ -0,0 +1,310 @@
+                        <span>Undo</span>
+                        <span class="material-icons  snackbar-icons" id="toastExampleRightCloseBtn">close</span>
+                    </div>
+
+
+                    <div class=" snackbar snackbar-danger flex flex-column  snack-fixed snack-center hide"
+                        id="toastExampleCenter">
+                        <span>This item already has the label "travel". You can add a new label.</span>
+                        <div class=" flex flex-justify-end  flex-align-item-center gap mt-1">
+                            <span>Undo</span>
+                            <span class="material-icons  snackbar-icons" id="toastExampleCenterCloseBtn">close</span>
+                        </div>
+                    </div>
+
+
+                </div>
+                <div class="card-footer">
+                    <script type="text/plain" class="line-numbers language-markup prism-default">
+                        <!-- Buttons -->
+                    <div class="flex flex-wrap flex-justify-center  flex-align-item-center gap">
+                        <button class="btn-primary btn btn-py-1" id="toastExample1">Toast left</button>
+                        <button class="btn-primary btn btn-py-1" id="toastExample2">Toast center</button>
+                        <button class="btn-primary btn btn-py-1" id="toastExample3">Toast right</button>
+                    </div>
+
+
+                    <!-- toasts -->
+                    <div class=" snackbar snackbar-primary flex flex-between flex-align-item-center gap snack-fixed snack-left hide"
+                        id="toastExampleLeft">
+                        <span>Can't send photo.Retry in 5 seconds.</span>
+                        <span>Retry</span>
+                        <span class="material-icons  snackbar-icons" id="toastExampleLeftCloseBtn">close</span>
+                    </div>
+
+                    <div class=" snackbar snackbar-success flex flex-between flex-align-item-center gap snack-fixed snack-right hide"
+                        id="toastExampleRight">
+                        <span>Order has been placed successfully.</span>
+                        <span>Undo</span>
+                        <span class="material-icons  snackbar-icons" id="toastExampleRightCloseBtn">close</span>
+                    </div>
+
+
+                    <div class=" snackbar snackbar-danger flex flex-column  snack-fixed snack-center hide"
+                        id="toastExampleCenter">
+                        <span>This item already has the label "travel". You can add a new label.</span>
+                        <div class=" flex flex-justify-end  flex-align-item-center gap mt-1">
+                            <span>Undo</span>
+                            <span class="material-icons  snackbar-icons" id="toastExampleCenterCloseBtn">close</span>
+                        </div>
+                    </div>
+
+
+                    </script>
```

Comment:
I think this is of no use. you can remove this.

Reply:
> I think this is of no use. you can remove this.

it's a closing tag to wrap code inside a script (Just a PrismJs Thing). I have wrapped all the HTML code inside the script tag.

Reply:
Ohh.. then it's ok..